gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/javaversions/BaselineJavaVersions.java

```diff
@@ -39,10 +34,14 @@ public final class BaselineJavaVersions implements Plugin<Project> {
     public static final String EXTENSION_NAME = "javaVersions";
 
     public static final GradleVersion MIN_GRADLE_VERSION = GradleVersion.version("7.0");
-    // 'nebula.maven-publish' and 'com.palantir.shadow-jar' create publications lazily which cause inconsistencies
-    // based on ordering.
+
     private static final ImmutableSet<String> LIBRARY_PLUGINS =
-            ImmutableSet.of("nebula.maven-publish", "com.palantir.shadow-jar", "com.palantir.external-publish-jar");
+            ImmutableSet.of("com.palantir.external-publish-jar", "com.palantir.publish-jar");
```

Comment:
I suspect we'll need to add some other plugins here eventually (possibly in a pluggable way :-s) but no need to implement that in the initial version.